### ISSUE TITLE
Customized run result not reasonable

### ISSUE THREAD

**Issue by lawy623** [facebookresearch/consistent_depth#6]
Thanks for sharing the code!

I run `python main.py --video_file $video_file_path --path $output_path --make_video`, with no calibrate camera. But the output results do not look reasonable. 

Any suggestion for this case? I see the finetuning makes disparity-0.1 and loss-0.5. Must I provided the calibrated intrinsics to run it well?

![image](https://user-images.githubusercontent.com/13533780/90615626-f71dda80-e23e-11ea-9b25-11feb2beaf01.png)

**Comment by roxanneluo** [comment]
Can you share your video? Larger baselines typically improves the results. Did you inspect the clomp reconstruction? Did the camera register correctly?

**Comment by lawy623** [comment]
I upload the video I filmed by iphone here: https://drive.google.com/file/d/1-iDkf5KWkxfdSp5hx2TqYhrc0q_6W7cj/view?usp=sharing

I inspect the depth_colmap_dense folder under the result directory and find this. Probably the reconstruction is not correct? 
![WeChatWorkScreenshot_c75335e4-5485-432e-a942-49075f7f0908](https://user-images.githubusercontent.com/13533780/90711398-3138bc00-e2d3-11ea-8cb9-6193bc6519f8.png)


I did not get into the code yet but I run with the default setting and change nothing except for setting the video_path.
I am not sure how should I make the reconstruction more reliable.
